shared/domain/src/commonMain/kotlin/network/bisq/mobile/client/websocket/rest_api_proxy/WebSocketRestApiClient.kt

```diff
@@ -0,0 +1,72 @@
+package network.bisq.mobile.client.websocket.rest_api_proxy
+
+import io.ktor.client.HttpClient
+import io.ktor.client.call.body
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import network.bisq.mobile.client.websocket.WebSocketClient
+import network.bisq.mobile.client.websocket.messages.WebSocketRestApiRequest
+import network.bisq.mobile.client.websocket.messages.WebSocketRestApiResponse
+import network.bisq.mobile.utils.Logging
+import kotlin.uuid.ExperimentalUuidApi
+import kotlin.uuid.Uuid
+
+class WebSocketRestApiClient(
```

Comment:
Are we implementing REST protocol over WS ? There is some strict nuances on naming and how to access resources and how to expose them. If we are then great, otherwise let's change the names removing REST prefix please 🙏